python/ray/rllib/env/vector_env.py

```diff
@@ -118,6 +122,20 @@ def get_unwrapped(self):
         return self.envs
 
 
+@ray.remote(num_cpus=0)
+class _RemoteEnv(object):
+    """Wrapper class for making a gym env a remote actor."""
+
+    def __init__(self, make_env, i):
+        self.env = make_env(i)
+
+    def reset(self):
+        return self.env.reset()
+
+    def step(self, action):
+        return self.env.step(action)
```

Comment:
Is it possible to call close() for remote env? SC2 environments are starting SC2 server which is a separate process, and and I guess the correct way to stop it in these situations would be calling the close method (though I see them dying after keyboard interrupt).

Reply:
There's python `atexit` which I think should work. If not, we can add close() hooks (but I don't know if this is as reliable in case of errors).